forge/files/src/index.js

```diff
@@ -59,9 +59,13 @@ app.on('ready', async () => {
     mainWindow.loadURL(emberAppURL);
   });
 
-  mainWindow.webContents.on('crashed', () => {
+  mainWindow.webContents.on('render-process-gone', (_event, details) => {
+    if (details.reason === 'killed' || details.reason === 'clean-exit') {
+      return;
+    }
```

Comment:
I think we should update the language here -- it kinda implies that it's a code problem, when it could be OOM or windows integrity or the user killing it intentionally with a SIGTERM, etc. Maybe we can just say `Your main window process has exited unexpectedly -- see <link to documentation for renderer-process-gone event>` and then also log out the reason.